server/game/cards/characters/06/ygritte.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Ygritte extends DrawCard {
+    setupCardAbilities(ability) {
+        // TODO: Cannot be knelt by card effects.
+        this.persistentEffect({
+            condition: () => this.controlsAnotherWildling(),
+            match: this,
+            effects: ability.effects.addKeyword('stealth')
+        });
+    }
+
+    controlsAnotherWildling() {
+        return this.controller.cardsInPlay.any(card => card.hasTrait('Wildling') && card.getType() === 'character' && card !== this);
```

Comment:
I've been moving away from directly operating on cardsInPlay recently in favour of a method on player, can we do that here?